bit_base_minions/windows/zabbix.sls

```diff
@@ -53,3 +53,22 @@ windows-retsart-zabbix-agent:
       - file: {{ psk_file }}
       - windows_zabbix_agent_conf
 
+
+
+
+           
+{%- if '_connection_url' in settings.zabbix %}
+update_zabbix:
```

Comment:
add the same to the linux one.. or rather a shared zabbix_update.sls that is included in windows and linux from the init.sls

Reply:
delete line 60 AND 61? or just 60 of zabbix.sls

Reply:
dont know exactly what you mean with that